app/lib/initializePlugins.js

```diff
@@ -1,12 +1,24 @@
 import { on, send } from 'lib/rpc'
 import plugins from 'plugins'
+import { settings as pluginSettings } from 'lib/plugins'
+
+export const getSettings = (name) => {
```

Comment:
I don't think `initializePlugins` is a good place for that. Could you move it? I think we can rename it to something like `getUserSettings` (that is more clear name) and move it to `lib/settings`

Reply:
Yes. Good point. I wanted to ask you about that. I'll see if I can move it right now.